tests/wrappers/ReactWrapper-test.js

```diff
@@ -191,5 +193,15 @@ describe('ReactInVue', () => {
         )
       )
     })
+
+    it('works with no children', () => {
```

Comment:
I think we should also check that there _is_ an error when we try to pass children to this component.

Reply:
Added a second test.